utils/api/caseStatus.ts

```diff
@@ -24,4 +24,4 @@ export const useCaseStatuses = (
 export const useFormValues = (
   type: string
```

Comment:
If there are only 3 possible types ("CIN, "CP" and "LAC") could we utilise an Enum here

Reply:
I'm not sure